schema_utils.py

```diff
@@ -215,9 +219,15 @@ def sanitize_url(obj):
           obj: a string representing a URL.
 
         Returns:
-          An empty string if the URL does not start with http:// or https://.
-          Otherwise, returns the original URL.
+          An empty string if the URL does not start with http:// or https://
```

Comment:
Here and elsewhere, we should indent by 4 not 2.

Reply:
Done.